templates/nfs-provisioner-deployment.yaml.j2

```diff
@@ -17,19 +17,23 @@ spec:
       serviceAccountName: nfs-client-provisioner
       containers:
         - name: nfs-client-provisioner
+{% if ansible_architecture == "ppc64le" %}
+          image: ibmcom/nfs-client-provisioner-ppc64le:latest
+{% else %}
           image: quay.io/external_storage/nfs-client-provisioner:latest
+{% endif %}
           volumeMounts:
             - name: nfs-client-root
               mountPath: /persistentvolumes
           env:
             - name: PROVISIONER_NAME
               value: nfs-storage
             - name: NFS_SERVER
-              value: {{ helper.ipaddr }}
+              value: {{ nfs_server | default(helper.ipaddr) }}
```

Comment:
IMHO this can be in a separate patch to add support for external NFS server and separate from using the right NFS client provisioner image per architecture. 
@christianh814 what do you think ?

Reply:
So in general, I like this approach because it keeps the default behaviour (which what I strive for) but gives the ability to extend it, if need be.

In general, as long as the default behaviour is preserved, I'm no too picky on the implementation.